### PR TITLE
Prevent double note

### PR DESCRIPTION
**Pull request by ArcaneDots**
Url hash handling method now blocks user from going directly to the "detail" page without setting a location marker. That marker and associated address objects are cleared once the POI is posted. The Url hash resets them immediately up loading the "done" page confirmation . 

- Fix for post call failure while running in a Firefox browser.
- Minor refactoring

### CODE REVIEW

File: js/site.js [hunk 1]
```diff
@@ -151,23 +154,23 @@ $("#find").submit(function (e) {
...
+          findMe_marker.setLatLng(activeMarkerLatLng);
         }
       });
 
       /**
        * Validate new marker location (Marker "drag ended" event)
        * @param {Object} dragged_event
        */
-      findme_marker.on('dragend', function (dragged_event) {
+      findMe_marker.on('dragend', function (dragged_event) {
```
Comment (iandees) [inline js/site.js]: Can you please undo the change to capitalization here? It's not necessary and is distracting from the actual logic changes below.

**Review by iandees — COMMENTED** [review]
Why rename the files?

**Comment by ArcaneDots** [comment]
The Italian fork added a second page (active.html) and mixed the assets together. I wanted to split them into separate trees. Each based on the html they are associated with. It will make it easier to manage any changes to files required by the libraries we use when they get upgraded. The problem is that most being use been considered deprecated.

**Comment by ArcaneDots** [comment]
Replacing with #128 which is a cleaner version of this PR
